### ISSUE TITLE
#import <NMSSH/NMSSH.h>: file not found

### ISSUE THREAD

**Issue by derBaecker94** [eugenehp/BonjourSSH#1]
Hi there,

I just downloaded your rep, but if I want to compile it, I see an error... #import <NMSSH/NMSSH.h>: file not found. Did you deleted a file?

Greetings,
Patrick

**Comment by eugenehp** [comment]
Hey Patrick,

That's the quickest resolution of GitHub issue I've ever seen 😁
Did you find the way that works for you?

Eugene

**Comment by derBaecker94** [comment]
Hi Eugene,

hahaha, I had just realised then about the dependency… Sorry for that. ;)
Actually not really, I have some Issues with the adding of NMSSH, but I found the example and will try that out later… I’m trying to make an app to connect myself to my raspberry pi, so your YouTube-Video was really the right one for me. Unfortunately I’m pretty new to iOS-Development and have therefore no much background with Swift, that’s kinda annoying at the moment.

But thanks for your Email, appreciate it :)

Greetings,
Patrick



On 17. Apr 2017, at 16:24, Eugene Hauptmann <notifications@github.com<mailto:notifications@github.com>> wrote:


Hey Patrick,

That's the quickest resolution of GitHub issue I've ever seen 😁
Did you find the way that works for you?

Eugene
